app/controllers/api/v1/projects_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class Api::V1::ProjectsController < ApplicationController
+  include JWTHelper
+  include ProjectsHelper
+
+  skip_before_action :verify_authenticity_token
+  before_action :authenticate_jwt
+
+  def create
+    save_project
+    create_code_location_subscription if @project.enlistments.exists?
+  rescue StandardError => e
+    render json: "Bad params #{e}", status: :bad_request
+  end
+
+  private
+
+  def project_params
+    params.require(:project).permit(
+      :name, :description, :vanity_url, :url, :download_url, :managed_by_creator, :coverity_project_id,
+      project_licenses_attributes: [:license_id],
+      enlistments_attributes: [code_location_attributes: %i[branch url scm_type]]
```

Comment:
* No branch, scm_type.
* Do not use nested attributes here. Expected a linear list of params and convert them in controller into nested attributes.
* Rename the first url to homepage_url.